src/Sql/Sql.Test/ScenarioTests/DataClassificationTests.ps1

```diff
@@ -0,0 +1,530 @@
+		
+		$recommendation = ($recommendations.SensitivityLabels)[0]
+		$schemaName = $recommendation.SchemaName
+		$tableName = $recommendation.TableName
+		$columnName = $recommendation.ColumnName
+		$informationType = $recommendation.InformationType
+		$sensitivityLabel = $recommendation.SensitivityLabel
+		
+		# Provide illegal information type, and verify error is raised.
+		$badInformationType =  $informationType + $informationType
+		$badInformationTypeMessage = "Information Type '" + $badinformationType + "' is not part of Information Protection Policy. Please add '" + $badinformationType + "' to the Information Protection Policy, or use one of the following: "
+		Assert-ThrowsContains -script { Set-AzSqlDatabaseSensitivityClassification -ResourceGroupName $params.rgname -ServerName $params.serverName `
+		-DatabaseName $params.databaseName -SchemaName $schemaName -TableName $tableName -ColumnName $columnName -InformationType $badInformationType `
+		-SensitivityLabel $sensitivityLabel} -message $badInformationTypeMessage
+
+		# Provide illegal sensitivity label, and verify error is raised.
+		$badSensitivityLabel = $sensitivityLabel + $sensitivityLabel
+		$badSensitivityLabelMessage = "Sensitivity Label '" + $badSensitivityLabel + "' is not part of Information Protection Policy. Please add '" + $badSensitivityLabel + "' to the Information Protection Policy, or use one of the following: "
+		Assert-ThrowsContains -script { Set-AzSqlDatabaseSensitivityClassification -ResourceGroupName $params.rgname -ServerName $params.serverName `
+		-DatabaseName $params.databaseName -SchemaName $schemaName -TableName $tableName -ColumnName $columnName -InformationType $badInformationType `
+		-SensitivityLabel $badSensitivityLabel} -message $badSensitivityLabelMessage
+		
+		# Do not provide neither information type nor sensitivity label, and verify error is raised.
+		$message = "Value is not specified neither for InformationType parameter nor for SensitivityLabel parameter"
+		Assert-ThrowsContains -script { Set-AzSqlDatabaseSensitivityClassification -ResourceGroupName $params.rgname -ServerName $params.serverName `
+		-DatabaseName $params.databaseName -SchemaName $schemaName -TableName $tableName -ColumnName $columnName} -message $message
+	}
+	finally
+	{
+		# Cleanup
+		Remove-DataClassificationTestEnvironment $testSuffix
+	}
+}
+
+function Assert-NotNullOrEmpty ($str)
+{
+	Assert-NotNull $str
+	Assert-AreNotEqual "" $str
+}
+
+<#
+.SYNOPSIS
+Gets the values of the parameters used at the tests
+#>
+function Get-DataClassificationTestEnvironmentParameters ($testSuffix)
+{
+	return @{ rgname = "dc-cmdlet-test-rg" +$testSuffix;
+			  serverName = "dc-cmdlet-server" +$testSuffix;
+			  databaseName = "dc-cmdlet-db" + $testSuffix;
+			  loginName = "testlogin";
+			  pwd = "testp@ssMakingIt1007Longer";
+		}
```

Comment:
Was the line an issue with Credential Scanner?

Reply:
No.